### ISSUE TITLE
Is the learning consistent between training in warped MS-COCO and UDIS-D?

### ISSUE THREAD

**Issue by DanilHu** [nie-lang/UnsupervisedDeepImageStitching#28]
Hello, could you please tell me: 
1.  Is the learning rate consistent after turning to UDIS-D or I have to reset it to 0.0001 just as the initial learning rate? And whether the dataset you use in `inference.py` and `out_inference.py` is just UDIS-D without inferencing in warped MS COCO? 

    ![q](https://user-images.githubusercontent.com/40864036/160612071-d47b561c-df72-47bb-91bc-0fa869259a79.png)

2. Why there are just 1106 masks you generated? Whether you just use 1106 images to train the reconstruction stage?

    ![q1](https://user-images.githubusercontent.com/40864036/160613716-2fb517d3-e021-4cbc-b9f9-a99f9aaf3a67.png)

**Comment by nie-lang** [comment]
1. a) I have not modified the learning rate in COCO or UDIS-D. Just keep it.
    b) UDIS-D. 
2. In out_inference.py, I generate the training set and testing set for the subsequent reconstruction stage. In particular, 10440 for training and 1106 for testing.

**Comment by DanilHu** [comment]
Woo~Thank you for your reply!
1. So the initial learning rates in COCO and UDIS-D are both 0.0001, and followd by the same exponential decay scheme?
2. About Q2(I find the code were somehow modified which causes this stupid question, sorry for that -_-).

**Comment by DanilHu** [comment]
Oh, and I have an idea about improving your work in Depth-aware but don't know if it can work or maybe you have tried this one but find it too difficult to do it? I'm considering if we can have a communication on emial or wechat？

**Comment by nie-lang** [comment]
Well, welcome to have a communication with me. ***@***.*** or wechat:18811752801)

2022-03-30 09:57:30DanilHu ***@***.***>写道：

Oh, and I have an idea about improving your work in Depth-aware but don't know if it can work or maybe you have tried this one but find it too difficult to do it? I'm considering if we can have a communication on emial or wechat？
